check-ntpoffset/lib/check-ntpoffset.go

```diff
@@ -189,9 +210,23 @@ func getNTPOffsetFromChrony() (offset float64, err error) {
 }
 
 func parseNTPOffsetFromChrony(out io.Reader) (offset float64, err error) {
+	var stratumErr error
 	scr := bufio.NewScanner(out)
 	for scr.Scan() {
 		line := scr.Text()
+		if strings.HasPrefix(line, "Stratum") {
+			flds := strings.Fields(line)
+			if len(flds) != 3 {
+				return 0.0, fmt.Errorf("failed to get ntp stratum")
+			}
+			var stratum int64
+			stratum, stratumErr = strconv.ParseInt(flds[2], 10, 64)
+			// Documents does NOT indicate about "Stratum : 0" means unsychronized...
```

Comment:
So `Stratum : 0` actually means that the computer is unsynchronized.
(I know, of course, stratum-0 should be the most reliable one, but it is OK because it is indeed "unsynchronized".)